sensor_msgs/PointCloud2 pcl1
sensor_msgs/PointCloud2 pcl2
---
geometry_msgs/Transform transform